showpass/Showpass-E2E-tests
# TA-38-Verify users ability to search for an event and then verify the event card details

**Description:**
There is a ‘Search events’ input field on the top bar of the ‘Home’ page. This allows the end users to look for a particular event by name and date.
This spec will be verifying that an event can still be found and that the event details properly show up on the event card.

Steps:
- Log into [Create Amazing Experiences](https://beta.showpass.com/) 
- Enter an event name into the search input field
- Click the search button ( the magnifying glass icon)
- Make sure the event card shows up
- Verify the event details on the event card

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-38

**How to test:**
Run the following spec and ensure it passes.
cypress/e2e/events/verify-events.cy.js

**Test execution results:**

<img width="1717" alt="TA-38-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/06ceb77b-4eb6-47ca-ab41-9046d0982d8f">
